src/pages/introducing-the-pact-property-checker/index.md

```diff
@@ -1,20 +1,54 @@
 ---
-title: Introducing the Pact analysis tools
+title: Introducing the Pact property checker
 date: 2018-05-17 13:16:01
 tags:
 ---
 
-Together with [Kadena](http://kadena.io/), [Monic](https://www.monic.co/) has developed new a static analysis tool for the [Pact](https://github.com/kadena-io/pact) smart contract language. In this post we'll talk about the purpose of our tool, what it can do today, and what we have planned for the future.
+Together with [Kadena](http://kadena.io/), [Monic](https://www.monic.co/) has developed new a static analysis tool for the [Pact](https://github.com/kadena-io/pact) smart contract language that we're calling the "property checker." In this post we'll talk about the purpose of our tool, what it can do today, and what we have planned for the future.
```

Comment:
Not a big deal, but if we're setting a precedent here I prefer "logical quotes".

Reply:
Cool -- didn't know there was a term for it! Apparently wikipedia uses logical style. This sounds good to me.

Reply:
I was always taught that _The Elements of Style_ is the only way, but it's painful to transition to that from the programming world.